### PR TITLE
Integration branch/menu

### PR DESCRIPTION
**Pull request by bkgc**
integrationBranch/menu Add menu storybook 
collaborators:  Bryan Kenny Gutierrez Conde
                          Alejandro Espejo Vacaflores

### CODE REVIEW

**Comment by bkgc** [comment]
image

**Comment by bkgc** [comment]
menu-collapsed
menu-normal

**Comment by olmedo90** [comment]
Captura de pantalla (834)

no es acorde al Figma.

**Comment by bkgc** [comment]
image
bug fixed

**Comment by bkgc** [comment]
image

File: src/ui/components/menu/MenuGlobal.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: #F7F8FC;
+    font-family: 'Mulish';
+    font-size: 18px;
+    padding: 0;
+    margin: 0;
+    margin-left: 50px;
```
Comment (Ditmar) [inline src/ui/components/menu/MenuGlobal.css:7]: please use rem

File: src/ui/components/menu/stories/MenuItem.stories.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { MenuItem } from "../wrapper/Menu/MenuItem/MenuItem";
```
Comment (Ditmar) [inline src/ui/components/menu/stories/MenuItem.stories.js:2]: use simple quote

**Review by Ditmar — COMMENTED** [review]
nice pr please fix all commets now

**Review by Ditmar — COMMENTED** [review]
nice pr! good job

**Pull request merged**